tests/platforms/linux/system/test_mixin__verify_system_python.py

```diff
@@ -10,40 +9,63 @@
 
 def test_valid_python3(monkeypatch, create_command):
     """If Briefcase is being run with the system python, verification passes."""
+    # /usr/bin/python3 exists
+    existing_python3 = MagicMock()
+    existing_python3.exists.return_value = True
+    monkeypatch.setattr(system, "Path", MagicMock(return_value=existing_python3))
 
-    # Mock the existence of a valid non-docker system Python
-    # with the same major/minor as the current Python
-    python3 = MagicMock()
-    python3.resolve.return_value = Path(
-        f"/usr/bin/python{sys.version_info.major}.{sys.version_info.minor}"
+    # mock the system Python's 'sys.version'
+    system_version = "3.13.3 (main, Aug 14 2025, 11:53:40) [GCC 14.2.0]"
+    system_python_check_output = MagicMock(return_value=f"{system_version}\n")
+    monkeypatch.setattr(
+        create_command.tools.subprocess, "check_output", system_python_check_output
     )
-    mock_Path = MagicMock(return_value=python3)
-    monkeypatch.setattr(system, "Path", mock_Path)
+
+    # mock the running Python's 'sys.version'
+    briefcase_version = "3.13.3 (main, Aug 14 2025, 11:53:40) [GCC 14.2.0]"
+    monkeypatch.setattr(create_command.tools.sys, "version", briefcase_version)
```

Comment:
No need to define the constant twice:
```suggestion
    monkeypatch.setattr(create_command.tools.sys, "version", system_versions)
```